.github/workflows/build.yml

```
@@ -60,7 +69,7 @@ jobs:
     - name: Smoke test
       run: |
         unzip asciidoctor-web-pdf-linux.zip -d out
-        ./out/asciidoctor-web-pdf ./out/examples/document/basic-example.adoc -o basic.pdf
+        ./out/asciidoctor-web-pdf-linux-${{ steps.extract_version.outputs.version }}/asciidoctor-web-pdf ./out/asciidoctor-web-pdf-linux-${{ steps.extract_version.outputs.version }}/examples/document/basic-example.adoc -o basic.pdf
```

We can use `${{ steps.package_info.outputs.version }}` no?


```suggestion
        ./out/asciidoctor-web-pdf-linux-${{ steps.package_info.outputs.version }}/asciidoctor-web-pdf ./out/asciidoctor-web-pdf-linux-${{ steps.package_info.outputs.version }}/examples/document/basic-example.adoc -o basic.pdf
```